src/components/Typography/Typography.styles.ts

```diff
@@ -1,5 +1,6 @@
 import { css } from 'styled-components';
 import color from '../../styles/colors';
+import fontBreakpoint from '../../styles/fontBreakpoint';
```

Comment:
Please update this import as well

Reply:
Oh, i already noticed the issue and pushed a fix for it 
Thanks